README.md

```diff
@@ -20,6 +20,7 @@ It is written with an aim to lazy load images, iframes, ads, videos or any other
 - [CSS Tricks](https://css-tricks.com/lozad-js-performant-lazy-loading-images)
 - [David Walsh](https://twitter.com/davidwalshblog/status/915319510646829061)
 - [Codrops](https://twitter.com/Apoorv_Saxena/status/906586828265758720)
+- [Digital Fortress](https://digitalfortress.tech/js/lazy-loading-images-vuejs/)
```

Comment:
you can remove this reference, as I've added a new page in the [Wiki](https://github.com/ApoorvSaxena/lozad.js/wiki/References) better suited for referencing the blog post

Reply:
yep, done.